src/widgets/Window.cpp

```diff
@@ -356,7 +356,7 @@ void Window::addShortcuts()
 
     // New tab
     createWindowShortcut(this, "CTRL+SHIFT+T", [this] {
-        this->notebook_->addPage(true);
+        auto *newPage = this->notebook_->addPage(true);
```

Comment:
```suggestion
        this->notebook_->addPage(true);
```

Reply:
No idea why was this changed, but sure why not.